ui/apps/ui/src/app/pages/acceptable-use-policy/acceptable-use-policy.component.html

```diff
@@ -0,0 +1,43 @@
+    <p>Users should read, understand and accept this AUP when creating an account on the Website or using any of the services provided via this Website. If a given user has not done this, or does not agree with the terms set out in this document, that user should not create an account on the Website and should not make use of any of the services provided via this Website.</p>
+    <p>By accepting this AUP, the registered User, enters into agreement with ACC Cyfronet AGH, made up by the contents of these terms, in order to regulate the user’s access and use of the Website. User who accept these AUP hereby represent and warrant that:</p>
+    <ol>
+      <li>You accept to be bound by these AUP, as registered Website user – either individually, or in representation of an organization;</li>
+      <li>You are capable, under the terms of the applicable law, to be legally bound by and comply with these AUP – in particular, whenever a you create an account as a representative of an organization, that you warrant to have been expressly and validly authorized by that organization to do so, or to otherwise be a legitimate representative of that organization under the applicable law;</li>
+      <li>You shall only use the Services in a manner consistent with the purposes and limitations described above; you shall show consideration towards other users including by not causing harm to the Services; you have an obligation to collaborate in the resolution of issues arising from your use of the Services.</li>
+      <li>You shall only use the Services for lawful purposes and not breach, attempt to breach, nor circumvent administrative or security controls.</li>
+      <li>You shall respect intellectual property and confidentiality agreements.</li>
+      <li>You shall protect your access credentials (e.g. passwords, private keys or multi-factor tokens); no intentional sharing is permitted.</li>
+      <li>You shall keep your registered information correct and up to date.</li>
+      <li>You shall promptly report known or suspected security breaches, credential compromise or misuse to the security contact stated below; and report any compromised credentials to the relevant issuing authorities.</li>
+      <li>Reliance on the Services shall only be to the extent specified by any applicable service level agreements listed below. Use without such agreements is at your own risk.</li>
+      <li>Your personal data will be processed in accordance with the <a href="https://docs.google.com/document/d/1Swi5XimmXH2XYHTZTXS4QU3oXUYlCP_ffqFhjydvxv0/edit">Privacy Policy of the EOSC User Dashboard</a>.</li>
```

Comment:
do we really want to pass google docs here?

Reply:
no, google docs replaced by angular routing to privacy-policy